documentation/modules/configuring/ref-list-of-kafka-cluster-resources.adoc

```diff
@@ -17,7 +17,8 @@ The following resources are created by the Cluster Operator in the Kubernetes cl
 
 .ZooKeeper nodes
 
-`_cluster-name_-zookeeper`:: StatefulSet which is in charge of managing the ZooKeeper node pods.
+`_cluster-name_-zookeeper`:: StatefulSet which is in charge of managing the ZooKeeper node pods (used only when xref:ref-operator-use-strimzi-pod-sets-feature-gate-{context}[`UseStrimziPodSets` feature gate] is disabled).
+`_cluster-name_-zookeeper`:: StrimziPodSet which is in charge of managing the ZooKeeper node pods (used only when xref:ref-operator-use-strimzi-pod-sets-feature-gate-{context}[`UseStrimziPodSets` feature gate] is enabled).
```

Comment:
why having two entries with the same name `_cluster-name_-zookeeper` instead of one providing the definition "if StrimziPodSet feature gate is enabled then it's the StrimziPodSet in charge ... else it's the StatefulSet in charge ...".
Two entries with the same name sound to be confusing to me.

Reply:
@ppatierno Any exact suggestion how would you phrase it? I tried something like

```
`_cluster-name_-zookeeper`:: Either StatefulSet which is in charge of managing the ZooKeeper node pods or StrimziPodset when when xref:ref-operator-use-strimzi-pod-sets-feature-gate-{context}[`UseStrimziPodSets` feature gate] is enabled.
```

But it seems weird and easy to miss when you just skim through the test. What you really see is the first part only.

Reply:
Tbh I like this one. If you skim through the text without reading everything, it's your fault not by who wrote the doc. Anyway it is a short text so it should be read in one go :-)

Reply:
> If you skim through the text without reading everything, it's your fault not by who wrote the doc. Anyway it is a short text so it should be read in one go :-)

Yeah, not really, because I (we) get that asked questions about the unclear things and that wastes our time.

Reply:
But two entries in the table could be more confusing at least it was for me. Le'ts ask the feedback from other maintainers, maybe it's just me.

Reply:
We have a few of these. Does presenting like this help?

![image](https://user-images.githubusercontent.com/47596553/152820665-7a6d999c-e310-41fd-b1bc-f08204ce35b0.png)

Reply:
We have a few of these. Does presenting like this help?

![image](https://user-images.githubusercontent.com/47596553/152820665-7a6d999c-e310-41fd-b1bc-f08204ce35b0.png)

Reply:
We have a few of these. Does presenting like this help?

![image](https://user-images.githubusercontent.com/47596553/152820665-7a6d999c-e310-41fd-b1bc-f08204ce35b0.png)

Reply:
We have a few of these. Does presenting like this help?

![image](https://user-images.githubusercontent.com/47596553/152820665-7a6d999c-e310-41fd-b1bc-f08204ce35b0.png)

Reply:
We have a few of these. Does presenting like this help?

![image](https://user-images.githubusercontent.com/47596553/152820665-7a6d999c-e310-41fd-b1bc-f08204ce35b0.png)

Reply:
We have a few of these. Maybe we could present like this.

![image](https://user-images.githubusercontent.com/47596553/152821808-123d9c0e-370d-46b9-ac50-0d2a5c71394e.png)

Reply:
We have a few of these. Maybe we could present like this.

![image](https://user-images.githubusercontent.com/47596553/152821808-123d9c0e-370d-46b9-ac50-0d2a5c71394e.png)

Reply:
Is it better now @ppatierno?

Reply:
Yeah I like it!